src/documents/ImageReordererAPI.md

```diff
@@ -0,0 +1,488 @@
+(empty string) </td>
+            <td>
+                <pre>
+                    <code>
+selectors: {
+    grabHandle: ".title-bar"
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+
+        <tr>
+            <td>`dropTargets`</td>
+            <td>Identifies the DOM elements contained within the Reorderer container that can have movable elements dropped relative to them. Note that not all elements within the container need to be drop targets.</td>
+            <td>same as movables</td>
+            <td>
+                <pre>
+                    <code>
+selectors: {
+    dropTargets: "div.dropTarget"
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+        <tr>
+            <td>`selectables`</td>
+            <td>Identifies the DOM elements contained within the Reorderer container that can be selected using keyboard. Note that selectable elements do not have to be movable. </td>
+            <td>same as movables</td>
+            <td>
+                <pre>
+                    <code>
+selectors: {
+    selectables: "div.selectable"
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+        <tr>
+            <td>`movables`</td>
+            <td>Identifies the DOM elements contained within the Reorderer container that can be moved using the Reorderer.</td>
+            <td>".flc-reorderer-movable"</td>
+            <td>
+                <pre>
+                    <code>
+selectors: {
+    movables: "div.movable"
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+        <tr>
+            <td>`dropWarning`</td>
+            <td>Identifies a single element within the DOM that can be shown to display a warning when the user tries to move an item where it can't be moved. It is assumed that this element contains whatever drop warning text and mark-up the implementor desires.</td>
+            <td>".flc-reorderer-dropWarning"</td>
+            <td>
+                <pre>
+                    <code>
+selectors: {
+    dropWarning: "#drop-warning"
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### Image Reorderer-specific Selector ###
+
+The Image Reorderer supports one additional selector:
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Description</th>
+            <th>Values</th>
+            <th>Default</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>`imageTitle`</td>
+            <td>Identifies a DOM element that contains a title for the image</td>
+            <td>`".flc-reorderer-imageTitle"`</td>
+            <td>
+                <pre>
+                    <code>
+selectors: {
+    imageTitle: ".caption"
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### General options ###
```

Comment:
It seems odd to put the general options before the specifics about selectors. Could we re-order these sections? So general first (then image-reorderer-specific), then selectors, then image-reorderer-specific selectors.